Registry:  Action history v2 (#1831)
wandb/docs: models/registry/lineage.mdx

---

> description: Use lineage graphs to visualize linked artifacts' history and audit a collection's history to track changes made to artifacts in that collection.

Reviewer: ```suggestion
description: Use lineage graphs to visualize a or audit a collection's linked artifact history to track changes made to that collection's artifacts.
```

> title: Lineage graphs and audit history

Reviewer: Howa bout something like "Use lineage graphs to visualize and audit artifact history" for more action-oriented?

---

Use a lineage graph to visualize a linked artifact's history. Audit a collection's history to track changes made to artifacts in that collection.

## Lineage graphs

Within a collection in the W&B Registry, you can view a history of the artifacts that an ML experiment uses. This history is called a _lineage graph_.

> A lineage graph shows:

Reviewer: ```suggestion
A lineage graph shows the input and output of a run:
```

* If a run used an [artifact as an input](/models/artifacts/explore-and-traverse-an-artifact-graph#track-the-input-of-a-run).

> * If a run created an [artifact as an output](/models/artifacts/explore-and-traverse-an-artifact-graph#track-the-output-of-a-run).

Reviewer: ```suggestion
* Whether the run used an [artifact as an input](/models/artifacts/explore-and-traverse-an-artifact-graph#track-the-input-of-a-run).
* Whether the run created an [artifact as an output](/models/artifacts/explore-and-traverse-an-artifact-graph#track-the-output-of-a-run).
```

> In other words, a lineage graph shows the input and output of a run.

Reviewer: ```suggestion
```

4. A run called "northern-lake-21" uses the model artifact `zoo-ylbchv20:v0` to evaluate the model.

To view a lineage graph for an artifact in a collection:

1. Navigate to the W&B Registry.
2. Select the collection that contains the artifact.
3. From the dropdown, select the artifact version you want to view its lineage graph.
4. Select the **Lineage** tab.
5. Select a node to view detailed information about the run or artifact.

<Info>

> See [Enable lineage graph tracking](/models/artifacts/explore-and-traverse-an-artifact-graph#enable-lineage-graph-tracking) in [Explore artifact lineage graphs](/models/artifacts/explore-and-traverse-an-artifact-graph) to learn how to track the input and output of a run using the W&B Python SDK.

Reviewer: It seems like the less specific link could be omitted.
